doc/content/modules/user-manual/pages/release-notes/2025.6.0.adoc

```diff
@@ -181,6 +181,8 @@ image::StateTransitionUsageLabels.png[TransitionUsage labels]
 image::release-notes-GV-actionDefinition-parameters.png[Parameters compartment on ActionDefinition]
 
 - In the _General View_ diagram, the selection dialog to reference an existing `Action` when creating a new `Perform Action` has been improved to present those `Actions` in a hierarchical way.
+- Make LibraryPackage's `isStandard` checkbox read-only in the details view.
```

Comment:
Make LibraryPackage's `isStandard` checkbox read-only in the details view. => Make LibraryPackage's `isStandard` checkbox read-only in the details view. => Make `LibraryPackage`'s `isStandard` checkbox read-only in the _Details_ view.